src/apiCalls.tsx

```diff
@@ -0,0 +1,33 @@
+export const getWeddings = () => {
+  return fetch('http://localhost:8000/api/v1/weddings/weddings/')
+  .then(response => {
+    if(response.ok) {
+      return response.json()
+    } else {
+      throw Error
+    }
```

Comment:
Super pumped to finally get a successful API call working

Reply:
Thank you @IamNorma @fieldstyler !